app/locale/en.coffee

```diff
@@ -963,7 +964,7 @@
     average_levels: "Average levels completed:"
     total_levels: "Total levels completed:"
     furthest_level: "Furthest level completed:"
-    concepts_covered: "Concepts Covered"
+    # concepts_covered: "Concepts Covered"
```

Comment:
Just delete i18n strings we won't be using any more if they're really going to be gone. The processing script may choke on irregularities in this file, so best to avoid creating any.